tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
My proposal was and is your later example;
```
@Foo => { 'A' => true, 'B' => true, 'C' => true}
@Bar => { 'A' => true, 'C' => false}
then {@Foo = true, @Bar = false} => {'B' => true}
```

So; `@A - @B` being; remove from `@A` all rules in `@B`, no matter how the rule is configured in `@B`.
This also solves this;
```
@Foo => { 'A' => true, 'B' => true, 'C' => true}
@Bar => { 'A' => true, 'D' => false}
```
then `@Foo - @Bar = {'B' => true, 'C' => true}` , i.e. `D` is _not_ enabled as result of subtracting a negative rule (i.e. `-- !== +`)
This would be consistent to;
```
@Foo => { 'A' => 'foo'}
@Bar => { 'A' => 'bar'}
```
then `@Foo - @Bar = {}` , which currently resolves to `{'A' => 'foo'}`

The last one is important, because if you want to disable rule `A` when using `@Foo` than currently it is not possible using _subtraction_ from another _set_. 
(There is other way; subtract the rule only)
So should one than _add_ a set where the rule is _negative_ (i.e. `false`; `+- = -`)?